vnext/local-cli/runWindows/utils/build.js

```diff
@@ -83,12 +83,22 @@ async function restoreNuGetPackages(options, slnFile, verbose) {
   ensureNugetSpinner.succeed('Found NuGet Binary');
 
   const msbuildTools = MSBuildTools.findAvailableVersion('x86', verbose);
-  await nugetRestore(
-    nugetPath,
-    slnFile,
-    verbose,
-    msbuildTools.installationVersion,
-  );
+  try {
+    await nugetRestore(
+      nugetPath,
+      slnFile,
+      verbose,
+      msbuildTools.installationVersion,
+    );
+  } catch (e) {
+    if (!options.isRetryingNuget) {
```

Comment:
Nit: could we avoid mutating the original object?

Reply:
TIL

---
In reply to: [405937870](https://github.com/microsoft/react-native-windows/pull/4536#discussion_r405937870) [](ancestors = 405937870)